lib/google_calendar.rb

```diff
@@ -1,33 +1,37 @@
+require 'googleauth'
+require 'google/apis/calendar_v3'
+
 class GoogleCalendar
   def initialize(config, logger)
     @config = config
 
     if @config['google']['key'].present?
-      @key = OpenSSL::PKCS12.new(@config['google']['key'], @config['google']['key_secret']).key
-    else
-      @key = Google::APIClient::PKCS12.load_key(@config['google']['key_file'], @config['google']['key_secret'])
+      # https://github.com/google/google-auth-library-ruby/issues/65
+      # https://github.com/google/google-api-ruby-client/issues/370
+      ENV['GOOGLE_PRIVATE_KEY'] = @config['google']['key']
```

Comment:
Sadly it doesn't seem to be avoidable. Looking through the Google code there doesn't seem to be a way to pass these values in directly.